dev_process.md

```diff
@@ -0,0 +1,144 @@
+AIRR Standards Development Process
+----------------------------------
+
+The airr-standards repository contains numerous sub-projects within it
+which are all highly interconnected. Our goal is a simple workflow for
+development that also insures stable versioning and releases for AIRR
+data and software associated with publications and database
+submissions. The current sub-projects include:
+
+* Minimum information about an Adaptive Immune Receptor Repertoire Sequencing Experiment (MiAIRR).
+
+* Specification of file formats:
+
+1. Alignment hit table
+
+2. Rearrangement annotations
+
+* OpenAPI definitions:
+
+1. MiAIRR objects
+
+2. Rearrangement annotations
+
+* Documentation for http://docs.airr-community.org
+
+* Python package for reading/writing file formats
+
+Some future sub-projects include:
+
+* Ontologies definitions for AIRR data elements.
+
+* Common Repository WG defining standard API with associated specification.
+
+* R package for file format reading/writing/validation.
+
+* Germline Database WG defining criteria, submission protocols, and
+  tools, with associated file formats and specifications.
+
+* Software WG defining standardized data sets and metrics.
+
```

Comment:
Everything up to here feels like it could live in README.md

Reply:
I agree, when we rework README and the documentation then we can move around. Seeing as I was in a writing mood, I wrote the stuff down now.